client/app/hosting/module/add/hosting-module-add.controller.js

```diff
@@ -19,6 +19,8 @@ angular
         path: '',
         minPasswdLength: 8,
         maxPasswdLength: 30,
+        minDBPasswordLength: 8,
```

Comment:
Putting this in a `.constants.js` file make more sense.

Reply:
OK, it's just a little fix, but I will put 
```
minPasswdLength: 8,
maxPasswdLength: 30,
minDBPasswordLength: 8,
maxDBPasswordLength: 31,
```
in a constants file.